pnpm-lock.yaml

```diff
@@ -3334,7 +3334,7 @@ packages:
       url-loader: 1.1.2_webpack@4.46.0
       vue: 2.7.8
       vue-loader: 15.10.0_pf2nacutanypibfndeigfabsry
-      vue-router: 3.5.4_vue@2.7.8
```

Comment:
Is this intentional and whats different?

Reply:
No, I only changed markdown-it and rand pnpm install

Reply:
What it means I dont know, gave it a quick google but did not find anything, my guess would be that it somehow has to do with the package it depends on or somehting like that.

@benmccann do you have any knowledge of this?

Reply:
I'm not exactly sure, but that change doesn't happen on my machine when I make the same change to `package.json` and run `pnpm install`. You can see what I get here: https://github.com/chartjs/Chart.js/pull/10638

I have pnpm 7.9.5. Maybe you have a different version?

Reply:
Interesting, i don't know will need to check that monday

Reply:
Seems version was the issue indeed, had 7.9.0 installed